ops-bedrock/docker-compose.yml

```diff
@@ -239,27 +239,10 @@ services:
       - ESPRESSO_SEQUENCER_CONSENSUS_SERVER_URL=http://consensus-server:8080
       - ESPRESSO_SEQUENCER_API_PORT=8080
       - ESPRESSO_SEQUENCER_STORAGE_PATH=/store/sequencer
-      - ESPRESSO_SEQUENCER_L1_USE_LATEST_BLOCK_TAG=true
       - ESPRESSO_SEQUENCER_L1_PROVIDER
       - RUST_LOG
     depends_on:
       orchestrator:
         condition: service_healthy
     extra_hosts:
       - "host.docker.internal:host-gateway"
-
-  commitment-task:
```

Comment:
I think we still want to run this for the devnet, no?

Reply:
I just reverted these changes - figured @sveitser will have the correct setup anyway in his PR